client/src/main/java/com/netflix/conductor/client/http/ClientBase.java

```diff
@@ -309,121 +219,221 @@ protected Map<String, Object> downloadFromExternalStorage(
         }
     }
 
-    private Builder getWebResourceBuilder(URI URI, Object entity) {
-        return client.resource(URI)
-                .type(MediaType.APPLICATION_JSON)
-                .entity(entity)
-                .accept(MediaType.TEXT_PLAIN, MediaType.APPLICATION_JSON);
+    private String getFullUrl(String url) {
+        return root + url;
     }
 
-    private void handleClientHandlerException(ClientHandlerException exception, URI uri) {
-        String errorMessage =
-                String.format(
-                        "Unable to invoke Conductor API with uri: %s, failure to process request or response",
-                        uri);
-        LOGGER.error(errorMessage, exception);
-        throw new ConductorClientException(errorMessage, exception);
+    private UriBuilder getURIBuilder(String path, Object[] queryParams) {
+        path = org.apache.commons.lang3.StringUtils.trimToEmpty(path);
```

Comment:
nit: fully qualified package name is not needed here

Reply:
done